pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -66,9 +68,63 @@ def set_start_date_from_period(self, period: int):
         start_dt = datetime.now() - timedelta(days=period) if int(period) > 0 else None
         self.start_date = start_dt
 
+    @enforce_types
+    def _check_cache_query_data(
+        self, query: str, cache_file_name: str, scalar: bool
+    ) -> Union[List[dict], pandas.DataFrame, None]:
+        """
+        Executes a query and caches the result in a parquet file for up to an hour.
+        If a cached file exists and is less than an hour old, the cached result is used.
+
+        Args:
+            query: SQL query to execute.
+            cache_file_name: Name of the cache file (without extension).
+            scalar: Boolean flag indicating if the result should be a scalar.
+
+        Returns:
+            Query result as a list of dictionaries (for scalar=False)
+            or a scalar value (for scalar=True),
+            or None if the query execution fails.
+        """
+        cache_file_dir = os.path.join(self.lake_dir, "exports", "cache")
+        cache_file_path = os.path.join(cache_file_dir, f"{cache_file_name}.parquet")
+
+        try:
+            # Ensure the cache directory exists
+            os.makedirs(cache_file_dir, exist_ok=True)
+
+            # Check if cache file exists and is less than 1 hour old
+            if os.path.exists(cache_file_path):
+                file_age = time.time() - os.path.getmtime(cache_file_path)
+                if file_age < 3600:  # 3600 seconds = 1 hour
```

Comment:
I would have this configurable :)

Reply:
Maybe we can create at some point a PPSS config for Dashboard, right now I don't know where to place this

Reply:
can you please make an issue? You can resolve this comment after pasting the link as a response :D

Reply:
I'm still thinking if creating a PPSS section for the dashboard it's needed. We only have this field for now.

Reply:
I have a better idea here. Since the new cache value is related to the new data exported to parquet files, instead of having a separate config param for caching, just use the already existent `seconds_between_parquet_exports`.
The cache update should be related to the new exported data anyway.